packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -298,8 +300,36 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
         _getListTile(
           appLocalizations.account_delete,
           () async {
+            await showDialog<SmoothAlertDialog>(
+                context: context,
+                builder: (BuildContext context) {
+                  return SmoothAlertDialog(
+                    title: appLocalizations.account_delete,
+                    body: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: <Widget>[
+                        const Text(
+                            'Are you sure you want to delete your account? \n Is there a specific reason share below '),
+                        const SizedBox(
+                          height: 5,
+                        ),
+                        SmoothTextFormField(
+                            type: TextFieldTypes.PLAIN_TEXT,
+                            textInputType: TextInputType.text,
+                            controller: reasonController,
+                            hintText: 'Reasons'),
+                      ],
+                    ),
+                    positiveAction: SmoothActionButton(
+                      text: appLocalizations.okay,
+                      onPressed: () => Navigator.maybePop(context),
+                    ),
+                  );
+                });
             final Email email = Email(
-              body: appLocalizations.email_body_account_deletion(userId),
+              body: appLocalizations.email_body_account_deletion(userId) +
```

Comment:
Please concatenate the strings using this annotation: "${var1} ${var2}"

Reply:
app_en.arb

Reply:
okay thanks !

Reply:
<img width="349" alt="translation" src="https://user-images.githubusercontent.com/71835085/200163107-1a1b1b55-0e02-4fa6-84a4-e1be55bf3542.png">

Tried to add to this file but cannot access it using appLocalizations.reason ?

Reply:
okay @g123k !!

Reply:
Hot reload / hot restart refresh the appLocalizations.

Reply:
Can someone tell from where to add translations ? @AshAman999 @g123k

Reply:
@monsieurtanuki I tried hot reload / restart but still can't access it

Reply:
I guess running `flutter pub get` should do the trick, too.

Reply:
Otherwise `flutter clean` and then running the app

What I do in some cases is just commiting with it not working locally. When the tests pass this passes as well

Reply:
Okay @monsieurtanuki @M123-dev !